__tests__/deleteUsers.test.js

```diff
@@ -0,0 +1,57 @@
+const name = "DeleteUsers";
```

Comment:
Can you also check on having unit test cases for deleteUsers around batching implementation ?
Ref: https://github.com/rudderlabs/rudder-transformer/pull/1537/files#diff-3107f146043efa1c7ff3cb2ba1ce699080ab88d2e4befc7a370e3bdede0bd1f4
Batching logic (the way data is chunked) should be moved to a separate function & have unit test case around it.

Reply:
I see from most of the implementations that we are using lodash's `chunk` method to batch the requests. If batching logic is what we are more concerned with then we'd be testing lodash's `chunk` method actually. Don't think it'd make sense. But if we want to verify if the mappings of userId are being done correctly, we can think on those lines. Not sure which one of these two is being referred to here, can you let me know @krishna2020 ?

Reply:
I meant the first one only & agree that we shouldn't be concerned as lodash's chunk is getting used & there's no additional custom logic around batching. At least for Mixpanel & Clevertap, no need at the moment.